compiler/crates/relay-compiler/src/errors.rs

```diff
@@ -17,6 +17,9 @@ pub type Result<T> = std::result::Result<T, Error>;
 
 #[derive(Debug, Error)]
 pub enum Error {
+    #[error("Unable to run the relay language server. Error details: \n{details}")]
+    LSPError { details: String },
```

Comment:
Rather than adding a new variant here, let's crate a new `errors.rs` file in `relay-bin`. It's okay if it just has an enum with one or two variants.